prometheus/testutil/testutil_test.go

```diff
@@ -300,34 +300,20 @@ func TestMetricNotFound(t *testing.T) {
 			"label1": "value1",
 		},
 	})
+
 	c.Inc()
 
 	expected := `
 		some_other_metric{label1="value1"} 1
 	`
 
-	expectedError := `
```

Comment:
We need to keep the form of this expected error. There is a reason that the original author put this here. They wanted to check for specific diff here.